Formula/wine.rb

```diff
@@ -65,49 +52,406 @@ def require_universal_deps?
   end
 
   resource "gecko" do
-    url "https://downloads.sourceforge.net/wine/wine_gecko-2.40-x86.msi", :using => :nounzip
-    sha256 "1a29d17435a52b7663cea6f30a0771f74097962b07031947719bb7b46057d302"
+    url "https://dl.winehq.org/wine/wine-gecko/2.47/wine_gecko-2.47-x86.msi", :using => :nounzip
+    sha256 "3b8a361f5d63952d21caafd74e849a774994822fb96c5922b01d554f1677643a"
   end
```

Comment:
you also need (something like) this:
```
resource "gecko-64" do
    url "https://dl.winehq.org/wine/wine-gecko/2.47/wine_gecko-2.47-x86_64.msi"
    sha256 "c565ea25e50ea953937d4ab01299e4306da4a556946327d253ea9b28357e4a7d"
end
```
For the 64 bit build of wine, both the x86 and x86_64 versions are needed and should be placed in the same "gecko" folder. See https://wiki.winehq.org/Gecko#Installing for details. 😄